src/client/templates/header_footer.html

```diff
@@ -34,7 +35,7 @@
     <!-- <hr /> -->
     <!--start of footer-->
         <div class="footor navbar navbar-fixed-bottom container-fluid">
-            <p class="text-muted text-center">&#169;2016 Zywave &amp; MSOE SDL HyperWave</p>
+            <p class="text-muted text-center">&#169;2015-2016 MSOE SDL HyperWave</p>
```

Comment:
No, bad!
https://github.com/Zywave/Retrospectre/issues/31
What PO says, PO gets
Including exact text, this was changed because he specifically requested it.